incubator/kafka/values.yaml

```diff
@@ -107,6 +107,49 @@ persistence:
   ##
   # storageClass:
 
+## Metrics
+##
+metrics:
+
+  jmx:
+    enabled: false
+
+    # The image to use for the metrics collector
+    image: solsson/kafka-prometheus-jmx-exporter@sha256
+
+    # The image tag to use for the metrics collector
+    imageTag: a23062396cd5af1acdf76512632c20ea6be76885dfc20cd9ff40fb23846557e8
+
+    # Rules to apply to the Kafka JMX Exporter
+    kafkaConfig:
+      lowercaseOutputName: true
+      jmxUrl: service:jmx:rmi:///jndi/rmi://127.0.0.1:5555/jmxrmi
+      ssl: false
+      whitelistObjectNames: ["kafka.server:*", "kafka.controller:*", "java.lang:*"]
+      # rules:
+      #   - pattern : kafka.server<type=ReplicaFetcherManager, name=MaxLag, clientId=(.+)><>Value
+      #   - pattern : kafka.server<type=BrokerTopicMetrics, name=(BytesInPerSec|BytesOutPerSec|MessagesInPerSec), topic=(.+)><>OneMinuteRate
+      #   - pattern : kafka.server<type=KafkaRequestHandlerPool, name=RequestHandlerAvgIdlePercent><>OneMinuteRate
+      #   - pattern : kafka.server<type=Produce><>queue-size
+      #   - pattern : kafka.server<type=ReplicaManager, name=(PartitionCount|UnderReplicatedPartitions)><>(Value|OneMinuteRate)
+      #   - pattern : kafka.server<type=controller-channel-metrics, broker-id=(.+)><>(.*)
+      #   - pattern : kafka.server<type=socket-server-metrics, networkProcessor=(.+)><>(.*)
+      #   - pattern : kafka.server<type=Fetch><>queue-size
+      #   - pattern : kafka.server<type=SessionExpireListener, name=(.+)><>OneMinuteRate
+      #   - pattern : kafka.controller<type=KafkaController, name=(.+)><>Value
+      #   - pattern : java.lang<type=OperatingSystem><>SystemCpuLoad
+      #   - pattern : java.lang<type=Memory><HeapMemoryUsage>used
+      #   - pattern : java.lang<type=OperatingSystem><>FreePhysicalMemorySize
+
+  kafka:
+    enabled: false
+
+    # The image to use for the metrics collector
+    image: danielqsj/kafka-exporter
+
+    # The image tag to use for the metrics collector
+    imageTag: latest
```

Comment:
Can we default to `v1.0.1` instead of latest, or is that too old?

Reply:
Looking at it, it doesn't appear that it's too old; just a few small bug fixes. I'll switch the default to `v1.0.1`, and will come back and bump the default when a new version is released 👍